### PR TITLE
Update privacy.html

### PR DESCRIPTION
**Pull request by GustavHemmelmayr**
changes for a YouTube credential

### CODE REVIEW

File: src/frontend/privacy.html [hunk 1]
```diff
@@ -953,6 +953,38 @@ <h2>Part II – Usage of personal information in SocialKYC</h2>
...
+          <ul>
+            <li>
+              SocialKYC asks the user to verify their YouTube account. The user 
+              does this by signing in with a YouTube account and channel, and authorizing 
+              us to request the name and ID of her channel from YouTube’s authentication service.</li>
+            <li>
+              The SocialKYC server requests the user’s channel info from YouTube, prepares a Request for 
+              attestation object for the user to sign and sends it to the user. This object contains the 
+              channel info properties but is not stored anywhere.
+            </li>
```
Comment (csmarc) [inline src/frontend/privacy.html:970]:
```suggestion
            <li>
                   The SocialKYC server requests the user’s channel info from YouTube and temporarily stores it 
                   in the server memory until the user successfully went through the complete credential flow, but 
                   no longer than 1 hour after the last user activity. The stored channel info contains the channel ID, 
                   channel name and the DID. Our server prepares a Request for attestation object including the 
                   channel info properties for the user to sign and sends it to the user. 
            </li>
```

File: src/frontend/privacy.html [hunk 2]
```diff
@@ -953,6 +953,38 @@ <h2>Part II – Usage of personal information in SocialKYC</h2>
...
+            <li>
+              Our server receives the signed Request for attestation object from the user and temporarily stores 
+              it until the user confirmation, but no longer than 1 hour after the last user activity. The object 
+              contains the channel ID, channel name and the DID.
+            </li>
+            <li>
+              If the user is inactive for one hour, all their stored data, the
+              username, and the confirmation code are deleted.
+            </li>
```
Comment (csmarc) [inline src/frontend/privacy.html:979]:
```suggestion
            <li>
              Our server receives the signed Request for attestation object from the user and also signs it. 
              Only the server’s signature is then stored on the public KILT blockchain, but the signature itself 
              does not contain any of the personal information, and this information is not recoverable from the 
              signature. After that all the information stored about this user is immediately deleted from the server.
            </li>
```

Comment (csmarc) [inline src/frontend/privacy.html:979]:
```suggestion
            <li>
              Our server receives the signed Request for attestation object from the user and also signs it. 
              Only the server’s signature is then stored on the public KILT blockchain, but the signature itself 
              does not contain any of the personal information, and this information is not recoverable from the 
              signature. After that all the information stored about this user is immediately deleted from the server.
            </li>
```

File: src/frontend/privacy.html [hunk 3]
```diff
@@ -953,6 +953,38 @@ <h2>Part II – Usage of personal information in SocialKYC</h2>
...
+            <li>
+              The server signs the stored Request for attestation object. Only the signature is 
+              then stored on the public KILT blockchain, but the signature itself does not contain 
+              any of the personal information, and this information is not recoverable from the signature. 
+              After that all the information stored about this user is immediately deleted from the server.
+            </li>
```
Comment (csmarc) [inline src/frontend/privacy.html:985]:
```suggestion
            <li>
              If the user is inactive for one hour, all their stored data, the properties, and the confirmation 
              code are deleted.
            </li>
```

Comment (csmarc) [inline src/frontend/privacy.html:985]:
```suggestion
            <li>
              If the user is inactive for one hour, all their stored data, the properties, and the confirmation 
              code are deleted.
            </li>
```

**Review by csmarc — CHANGES_REQUESTED** [review]
we should change these

**Review by csmarc — APPROVED** [review]
now it is almost perfect

**Review by arty-name — APPROVED** [review]

**Pull request merged**